README.md

```diff
@@ -67,12 +67,12 @@ Asserts that the target is False:
 expect(False).to.be.false()
 ```
 
-#### instanceof
+#### instance_of
```

Comment:
Pycharm keeps yelling that there is a grammatical mistake there. Moreover, we have `respond_to`, `called_with` so `instance_of` will be more suitable.